price-fetcher/app.ts

```diff
@@ -0,0 +1,15 @@
+
+function priceFetch( nfts: any[]): string[] {
+	
+	const prices: string[]
```

Comment:
I was thinking of converting prices to strings and just returning them from this function.  Give me some design feedback but I asssume thats all this function will do.

aka:
Grab prices,
Put them in an array,
return that array

done